src/main/java/core/basesyntax/MyLinkedList.java

```diff
@@ -3,45 +3,164 @@
 import java.util.List;
 
 public class MyLinkedList<T> implements MyLinkedListInterface<T> {
+
+    private Node<T> first;
```

Comment:
Don't use class Objects. Instead, make `Node<T>` an inner class to group classes that belong together, which makes your code more readable and maintainable.